### PR TITLE
Add spacing markup to non component files

### PR DESCRIPTION
**Pull request by araiford**
This PR adds spacing markup to markdown files that are not component docs.  @nimbinatus will be adding spacing to component markdown files.

### CODE REVIEW

**Comment by nimbinatus** [comment]
@technabors, that's just so I can test it while I'm building. I'll be deleting my class css from _layouts in my PR.

File: _docs/getting-started/design.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ last-modified: 2017-10-24
 
 
 
-<section>
+<section class="static-section">
```
Comment (nimbinatus) [inline _docs/getting-started/design.md]: @araiford, is yours building ok without `markdown="1"` in here? Haven't tried building this yet, but I had to add that to get my stuff to render right on the other PR.

Comment (nimbinatus) [inline _docs/getting-started/design.md]: These ones are the only ones without it, it looks like, so I'd add them in here.

Comment (araiford) [reply _docs/getting-started/design.md]: @nimbinatus -- nope.  I had to add markdown="1".  If I missed it on a couple, I'll add it.

**Comment by araiford** [comment]
@nimbinatus -- updated per your notes

**Pull request merged**